src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,6 @@
-const booleanToWord = (boolean) => {
-
-}
+const booleanToWord = (word) => {
```

Comment:
I don't think this would work if I passed it a 'no'. Try adding another test and trying to pass it.